modules/psap-installing-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-node-feature-discovery-operator.adoc
+
+[id="installing-the-node-feature-discovery-operator_{context}"]
+= Installing the Node Feature Discovery Operator
+
+The Node Feature Discovery (NFD) Operator manages detection of hardware features and configuration in a kubernetes cluster. The Operator orchestrates all resources needed to run the Node-Feature-Discovery daemon set As a cluster administrator, you can install the NFD Operator using the {product-title} CLI or the web console.
```

Comment:
"The Node Feature Discovery (NFD) Operator manages detection of hardware features and configuration in a kubernetes cluster." - this sentence is a repeat of the first sentence in "About the Node Feature Discovery Operator" section.